tools/graphite.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Code looks good to me, but you may want to add 'rhel' to the file name just to make it clear that its only meant for rhel/centos.